docs/src/architecture/transactions.md

```diff
@@ -1,45 +1,49 @@
 # Transactions
-Asset transfers between accounts are done by executing transactions. Miden aims for parallel and private transaction execution. Therefore, a transaction is always performed against a single account which provides asynchronicity. And, every transaction causes a provable state-change and a STARK proof thereof. Thus, when executed locally, it provides privacy.
+Transactions in Miden can be understood as facilitating account state changes. Asset transfers between accounts are done by executing transactions. Basically, they take one single account and some notes as input and output the same account at a new state together with some other notes. Miden aims for parallel and private transaction execution. Because a transaction is always performed against a single account Miden obtains asynchronicity. And, because every transaction causes a provable state-change and a STARK proof thereof it provides privacy, when executed locally.
 
 ## Transaction design
-Transactions can be described as state-transition function T that takes an account and `0 to n` notes to map it to another version of that account and produces `0 to n` notes
-
-$
-T(A, \sum_{\substack{
-   0 \leq i < 1024
-  }}
- N(i))) \to (A', \sum_{\substack{
-   0 \leq j < 4096
-  }}
- N(j))
-$
-, where  $ A \in { Accounts }$, $N \in { Notes }, 0 < i + j  $
```

Comment:
This doesn't render on Github. Should we simplify it?

Reply:
I have removed it now. I think it doesn't really add value